Lib/test/test_tempfile.py

```diff
@@ -1460,6 +1460,16 @@ def test_explict_cleanup_ignore_errors(self):
                 temp_path.exists(),
                 f"TemporaryDirectory {temp_path!s} exists after cleanup")
 
+    @unittest.skipUnless(os.name == "nt", "Only on Windows.")
+    def test_explicit_cleanup_correct_error(self):
+        with tempfile.TemporaryDirectory() as working_dir:
```

Comment:
Please add # Issue #43153 here too.

Reply:
Actually issue can be found by looking it up via github PR, I've seen these comments on unit tests, but maybe they're not necessary.

Reply:
Sure!

I've seen many arguments for and against that. Some people say we don't need it because we have git history, while others say CPython has changed version control system 3 times :). Personally, I add them if a piece of code looks awfully strange to save the next reader some time, and this PR definitely qualifies.

I wonder how things like `#Issue #43153` will hold up too, considering we're moving to GH issues soon, and the GH issue numbers will differ from the BPO ones. Hopefully `bpo-43153` is enough.